reader.js

```diff
@@ -7,15 +9,156 @@ const CLOSE_TIMEOUT = 10000;
 
 let WSS;
 
-function init(wss) {
+function init(server) {
     console.log('Initializing nasonov-reader');
+    const wss = new WebSocket.Server({server});
```

Comment:
```suggestion
    const wss = new WebSocket.Server({ server });
```

Style nit; we should add an automated style checker for nasonov at some point.